go.mod

```diff
@@ -6,9 +6,9 @@ require (
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/opencontainers/runc v1.0.3
 	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
-	github.com/opencontainers/runtime-tools v0.0.0-20190417131837-cd1349b7c47e
+	github.com/opencontainers/runtime-tools v0.9.0
```

Comment:
This change will break builds. We use `AddMultiplePorcessEnv()` from runtime-tools which is not part of any tagged release, so it requires a SHA1-exact dependency.

Reply:
Of course, if this is a show-stopper then we can rework CDI to use a much older version and later switch back to using `AddMultipleProcessEnv`, if that ever gets covered by a new tag.

Reply:
> Not sure it will be ever covered by a new tag.

Hmm, now I'm not sure what you mean. Do you mean that it is possible that there will never be any newer tagged version of `runtime-tools` than the current `0.9.0` ? Wouldn't that sort of imply that there would never be any future updates to the `OCI Spec/runtime-spec package` ?